<launch>
    <param name="num_robots" value="4" type="int"/>

    <arg name="launch_path" value="$(find modquad_simulator)/launch/aux/"/>


    <include file="$(arg launch_path)/modquad.launch">
        <arg name="model" value="modquad01"/>

        <arg name="init_x" default="0.0"/>
        <arg name="init_y" default="0.0"/>


        <arg name="color_r" default="0.0"/>
        <arg name="color_g" default="1.0"/>
        <arg name="color_b" default="0.0"/>
    </include>
    <include file="$(arg launch_path)/modquad.launch">
        <arg name="model" value="modquad02"/>

        <arg name="init_x" default="1.0"/>
        <arg name="init_y" default="0.0"/>


        <arg name="color_r" default="0.0"/>
        <arg name="color_g" default="1.0"/>
        <arg name="color_b" default="0.0"/>
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad03"/> 
 
        <arg name="init_x" default="1.0"/> 
        <arg name="init_y" default="1.0"/> 

 
        <arg name="color_r" default="0.0"/>
        <arg name="color_g" default="1.0"/>
        <arg name="color_b" default="0.0"/>
    </include>

    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad04"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="1.0"/> 
        <arg name="color_g" default="0.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad05"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.0"/> 
        <arg name="color_g" default="1.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad06"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.0"/> 
        <arg name="color_g" default="1.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad07"/> 
 
        <arg name="init_x" default="1.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.0"/> 
        <arg name="color_g" default="1.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad08"/> 
 
        <arg name="init_x" default="1.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.0"/> 
        <arg name="color_g" default="1.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad09"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.00"/> 
        <arg name="color_g" default="1.00"/> 
        <arg name="color_b" default="0.00"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad10"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.00"/> 
        <arg name="color_g" default="1.00"/> 
        <arg name="color_b" default="0.00"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad11"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.00"/> 
        <arg name="color_g" default="1.00"/> 
        <arg name="color_b" default="0.00"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad12"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.00"/> 
        <arg name="color_g" default="1.00"/> 
        <arg name="color_b" default="0.00"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad13"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.0"/> 
        <arg name="color_g" default="1.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad14"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.0"/> 
        <arg name="color_g" default="1.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad15"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.0"/> 
        <arg name="color_g" default="1.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>
    <include file="$(arg launch_path)/modquad.launch"> 
        <arg name="model" value="modquad16"/> 
 
        <arg name="init_x" default="0.0"/> 
        <arg name="init_y" default="1.0"/> 
 
 
        <arg name="color_r" default="0.0"/> 
        <arg name="color_g" default="1.0"/> 
        <arg name="color_b" default="0.0"/> 
    </include>

    <param name="num_robots" value="5" type="int"/>
    <!--Robot Visualization-->
    <!--<node pkg="modquad_simulator" type="visualizer.py" name="visual"/>-->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find modquad_simulator)/rviz/thirteen-robots.rviz" required="true"/>


    <!--<node pkg="dock_manager" type="docking_detector.py" name="docking_det"/>-->
    <!--<node pkg="modquad_simulator" type="docking_sim.py" name="docking_sim"/>-->
    <!--<node pkg="dock_manager" type="odom_multiplexer.py" name="odom_mux"/>-->


    <node pkg="tf2_ros" type="static_transform_publisher" name="link1_broadcaster"
          args="1 0 0 0 0 0 1 world link1"/>

</launch>
